src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/StoredProcedures/Upsert/upsertWithHistory.js

```diff
@@ -15,25 +15,17 @@ function upsertWithHistory(doc, matchVersionId, allowCreate, keepHistory) {
     const collectionLink = collection.getSelfLink();
     const response = getContext().getResponse();
 
-    const errorCodes = { ServiceUnavailable: 503 };
-
-    const errorMessages = {
-        DocumentNull: `${ErrorCodes.BadRequest}: The document is undefined or null.`,
-        InputWasArray: `${ErrorCodes.BadRequest}: Input should not be an array.`,
-        RequestNotQueued: `${errorCodes.ServiceUnavailable}: Request could not be queued.`,
-        DocumentNotFound: `${ErrorCodes.NotFound}: Document not found.`,
-        PreconditionFailed: `${ErrorCodes.PreconditionFailed}: One of the specified pre-condition is not met.`
-    };
+    
 
     const initialVersion = "1";
 
     // Validate input
     if (!doc) {
-        throw new Error(errorMessages.DocumentNull);
+        throw new Error(ErrorCodes.BadRequest, "The document is undefined or null.");
```

Comment:
nit: This could be a function that takes the message so the creation of errors is the same throughout the stored procedure.